docker/Dockerfile

```diff
@@ -55,7 +55,7 @@ RUN \
 # NB: In poetry 1.2 `poetry export` will be moved into a plugin; we'll need to also
 # pip install poetry-plugin-export (https://github.com/python-poetry/poetry-plugin-export).
 RUN --mount=type=cache,target=/root/.cache/pip \
-  pip install --user git+https://github.com/python-poetry/poetry.git@fb13b3a676f476177f7937ffa480ee5cff9a90a5
+  pip install --user "poetry-core==1.1.0a7" "git+https://github.com/python-poetry/poetry.git@fb13b3a676f476177f7937ffa480ee5cff9a90a5"
```

Comment:
I remember fighting with this a lot, though I can't exactly remember what this commit did for us. Probably a case of "latest master fixed it" and this was the commit at the time.

I also remember that after I updated some dependency specifications in signedjson that there were suddenly fewer bugs with the export. Given that the debian image doesn't use this commit, I wonder if it might be worth trying to use the same version that the docker image uses.